CardDictionaries/ArcaneRising/ARCWizard.php

```diff
@@ -354,6 +364,9 @@ function ArcaneDamage($cardID)
       case "DYN199": case "DYN204": case "DYN207": return 2;
       case "DYN205": case "DYN208": return 1;
       case "HVY252": return 1;
+      case "ROS207": case "ROS176": case "ROS189": case "ROS198": case "ROS201": return 3;
+      case "ROS208": case "ROS177": case "ROS190": case "ROS199": case "ROS202": return 2;
+      case "ROS209": case "ROS178": case "ROS191": case "ROS200": case "ROS203": return 1;
```

Comment:
for these I would just recommend that the individual lines be sorted in ascending set number. obviously sorting these perfectly would require way too many extra lines.